src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -494,43 +463,27 @@ private string GetOfflinePackagePath(SdkFeatureBand sdkFeatureBand, ManifestId m
             return packagePath;
         }
 
-        private async Task<(bool, string)> GetManifestPackageUpdate(SdkFeatureBand sdkFeatureBand, ManifestId manifestId, bool includePreviews, DirectoryPath? offlineCache = null)
+        private async Task<(bool Success, string PackagePath)> GetManifestPackageUpdate(SdkFeatureBand sdkFeatureBand, ManifestId manifestId, bool includePreviews, DirectoryPath? offlineCache = null)
```

Comment:
Always name the elements in a `ValueTuple`. Otherwise, no one knows what is their purpose without digging through the references.